site/content/blog/package-authoring-with-kctrl.md

```diff
@@ -0,0 +1,56 @@
+
+
+## Typical Package Authoring journey
+Package Authoring is an iterative process and below are the most common steps performed by the authors:
+1. Authors know about the Kubernetes manifest they want to package.
+2. Add/change the manifest by adding additional [overlay](https://carvel.dev/ytt/docs/latest/ytt-overlays/)/[template](https://carvel.dev/ytt/docs/latest/#templating) and test the package. This is the iterative part where authors want to make the changes and test them quickly.
+3. Once all the manifest are in place, create the [imgpkg](https://carvel.dev/imgpkg/) bundle (to be mentioned in the package) and the package itself.
+4. Add the package to the package repository for distribution.
+
+## How does `kctrl` help package authors?
+Today, package authors are supposed to know tools like [imgpkg](https://carvel.dev/imgpkg/), [kbld](https://carvel.dev/kbld/), [vendir](https://carvel.dev/vendir/), etc. before they start on the package authoring journey. These Carvel tools has a learning curve of itself. We wanted to introduce a set of CLI commands that guide users in performing most common packaging steps so that they are able to create and release the package without knowing these tools.
+
+{{< asciinema key="authoring-commands-blog" rows="30" preload="1" speed="2">}}
+
+## What are `kctrl` package authoring commands?
+* [**kctrl package init**](/kapp-controller/docs/latest/authoring-command/#initialising-the-package): To initialize the Package, PackageInstall and other resources. These resources will be used by the subsequent commands.
+
+* [**kctrl dev**](/kapp-controller/docs/latest/authoring-command/#dev): This command will use the Package and PackageInstall generated above and deploy them locally. By locally, we mean that kapp-controller need not be installed on the Kubernetes cluster. Also, it will eliminate the need to push the imgpkg bundle to an OCI registry during the development stage. This will be useful when you are develping the additional overlay/template.
+
+* [**kctrl package release**](/kapp-controller/docs/latest/authoring-command/#releasing-the-package): This command will create and upload the imgpkg bundle with all the Kuberentes manifest of the software. Also, it will create the `package.yml` and `metadata.yml` which can be either checkin to the package repository or released as part of the release artifacts.
+
+* [**kctrl package repo release**](/kapp-controller/docs/latest/authoring-command/#releasing-a-package-repository): This command will create and push the package repository bundle consisting of all the package and packageMetadata files present in the `packages` folder in the working directory. This repository bundle can later on be consumed by package consumers.
```

Comment:
package => Package
packageMetadata => PackageMetadata
in the working directory => of the working directory

Reply:
I would insist on naming the resources appropriately as well